inputs/templates/terra_workspaces/cohort_mode/workflow_configurations/output_configurations/GatherBatchEvidenceOutputs.json.tmpl

```diff
@@ -30,6 +30,7 @@
   "GatherBatchEvidence.std_delly_vcf": "${this.std_delly_vcf}",
   "GatherBatchEvidence.std_manta_vcf": "${this.std_manta_vcf}",
   "GatherBatchEvidence.std_melt_vcf": "${this.std_melt_vcf}",
+  "GatherBatchEvidence.std_scramble_vcf": "${this.std_scramble_vcf}",
```

Comment:
This seems like an unrelated change not mentioned in the PR description -- just wanted to make sure it's intentionally added here.

Reply:
Yeah I just noticed it while I was working on this branch and didn't want to make a whole separate PR for this one line